azure-functions/acbs-function/package.json

```diff
@@ -32,7 +32,9 @@
     "date-fns": "^2.30.0",
     "dotenv": "^16.4.5",
     "durable-functions": "^3.1.0",
-    "lodash": "^4.17.21"
+    "immer": "^10.1.1",
```

Comment:
Please remove newly introduced dependencies, the existing dependencies will suffice any implementation.